frontend/common/crud/useGet.tsx

```diff
@@ -23,12 +27,21 @@ export function useGet<T>(
   if (error && !(error instanceof Error)) {
     error = new Error('Unknown error');
   }
+
   return useMemo(
     () => ({
       data: response.data,
       error: response.isLoading ? undefined : error,
       refresh,
       isLoading: response.isLoading,
+      /**
+       * Renders error or loading element, can be used to simplify loading and error logic in application.
+       */
+      renderErrorOrLoading: error ? (
+        <AwxError error={error} handleRefresh={refresh} />
+      ) : response.isLoading ? (
+        <LoadingPage breadcrumbs tabs />
+      ) : null,
     }),
```

Comment:
@jamestalton please, can you review those changes?

Reply:
I'm not wild about the useGet which currently only returns data, returning a function that returns react components. But I do see the benefits of it and it solves the problems of consistency in the pages for loading and error handling.

While the name `renderErrorOrLoading` is more descriptive, it would read clearer in the pages if it was `loadingOrError`.
```
if( loadingOrError ) return loadingOrError
```

Reply:
Ok, I will do it old way without any changes in useGet. We will discuss this in future.

Reply:
I'm not against it. Like I said, I do see the benefits it provides.